client/blocks/taxonomy-manager/list-item.jsx

```diff
@@ -21,9 +21,11 @@ class TaxonomyManagerListItem extends Component {
 		translate: PropTypes.func,
 		onClick: PropTypes.func,
 		onDelete: PropTypes.func,
+		isDefault: PropTypes.bool,
```

Comment:
yes, but default is a reserved word, that's why I used `isDefault`. I guess I can quote it and avoid props deconstructing.